lib/pavilion/plugins/commands/status.py

```diff
@@ -36,6 +41,7 @@ def status_from_test_obj(pav_cfg, test_obj):
             'state': status_f.state,
             'time': status_f.when.strftime("%d %b %Y %H:%M:%S %Z"),
             'note': status_f.note,
+            'scheduler info': scheduler_status
```

Comment:
This is not ok. 

You need to read the documentation on this object more. Output is limited to the size of a block to ensure the file append is atomic. 

Adding to this object is a pretty major change, and do we need to know this information at every stage?

This makes me think that maybe this whole concept should be folded into the general scheduler status, and then incorporated into the status note in those situations.